portal/src/main/resources/sample.application.conf

```diff
@@ -1,3 +1,18 @@
+webservice {
+  interface = "<host_address>",
+  port = <port_number>
+}
+
+auth {
+  secretKey = "<secret_key>",
+  hmacAlgorithm = "<HMAC_algorithm>",
+  expirationTimeInSeconds {
```

Comment:
Use `FiniteDuration` instead of `Long`. Then you'll be able to call `.toSeconds` method to get amount of seconds.

Reply:
I liked your suggestion but after some investigation, I found out it inconvenient for use in terms of config reading and constant converting values from requests and to responses.